xbmc/LangInfo.cpp

```diff
@@ -1133,6 +1134,11 @@ void CLangInfo::SettingOptionsStreamLanguagesFiller(const CSetting *setting, std
 
   std::string dummy;
   SettingOptionsISO6391LanguagesFiller(NULL, list, dummy, NULL);
+  SettingOptionsLanguageNamesFiller(NULL, list, dummy, NULL);
+  std::set<std::pair<std::string, std::string>> languages;
+  unsigned size = list.size();
+  for (unsigned i = 0; i < size; ++i) languages.insert(list[i]);
```

Comment:
Manual loop because of http://stackoverflow.com/questions/1041620/whats-the-most-efficient-way-to-erase-duplicates-and-sort-a-vector . But if you still prefer the range version I change it.